.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "googletest"]
```

Comment:
This should probably be added at a specific version/tag (and documented as such), rather than whatever was latest